ansible/tljh.yml

```diff
@@ -24,18 +24,25 @@
       changed_when: False
 
     - name: Upgrade the tljh-plasmabio plugin first if it is already installed
-      shell: "{{ tljh_prefix }}/hub/bin/pip3 install --upgrade {{ tljh_plasmabio }}"
+      shell: "{{ tljh_prefix }}/hub/bin/pip3 install --upgrade {{ tljh_plasmabio }} {{ tljh_repo2docker }}"
       when: tljh_plasmabio_installed.rc == 0
 
     - name: Run the TLJH installer
-      shell: "{{ ansible_python_interpreter }} {{ tljh_installer_dest }} --no-user-env --plugin {{ tljh_plasmabio }}"
+      shell: |
+        {{ ansible_python_interpreter }} {{ tljh_installer_dest }} --no-user-env \
+          --plugin {{ tljh_plasmabio }} {{ tljh_repo2docker }}
       # TODO: remove when --no-user-env (or equivalent) is available
       environment:
         TLJH_BOOTSTRAP_PIP_SPEC: "{{ tljh_bootstrap_pip_spec }}"
 
     - name: Set the idle culler timeout to 1 hour
       shell: "tljh-config set services.cull.timeout 3600"
 
+    - name: Set the default memory and cpu limits
+      shell: |
+        tljh-config set limits.memory 2G
+        tljh-config set limits.cpu 2
```

Comment:
We can now set the default limits using `tljh-config`, which is better than having them hardcoded in the Python code of the plugin.

The `tljh-repo2docker` plugin is then able to pick them up to display the defaults in the UI: https://github.com/plasmabio/tljh-repo2docker/blob/ec9e2dee20095b11b4a933915694178775d50611/tljh_repo2docker/__init__.py#L80-L84